clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/core/services/Front50Service.groovy

```diff
@@ -35,9 +33,6 @@ interface Front50Service {
   Map getProject(@Path('project') String project)
 
   @GET('/v2/projects/search')
```

Comment:
Maybe I'll have Rob (when he starts next week) support searching via `/v2/projects` with query parameters vs. `/v2/projects/search`. That would make it a _little_ more consistent with how application search is done.